modules/ossm-vs-istio.adoc

```diff
@@ -93,4 +93,4 @@ spec:
 = Envoy, Secret Discovery Service, and Certificates
 
 * The QUIC listener for Envoy is not available in {ProductName}. 
```

Comment:
I wonder if this is a bit too cryptic? Perhaps something along the lines of "ServiceMesh doesn't support termination of QUIC-based services"? Or even simpler "ServiceMesh doesn't support QUIC-based services"?